command/elevator.py

```diff
@@ -0,0 +1,43 @@
+import utils
+import constants
+
+from toolkit.command import SubsystemCommand
+from subsystem import Elevator
+
+class ZeroElevator(SubsystemCommand[Elevator]):
+        
+        def __init__(self, subsystem: Elevator):
+            super().__init__(subsystem)
+    
+        def initialize(self):
+            self.subsystem.zero()
+    
+        def execute(self):
+            pass
+    
+        def isFinished(self):
+            utils.LocalLogger.debug("Elevator zeroed: " + str(self.subsystem.zeroed), "ZeroElevator")
+            return self.subsystem.zeroed
+
+class SetElevator(SubsystemCommand[Elevator]):
+    
+    def __init__(self, subsystem: Elevator, length: float):
+        super().__init__(subsystem)
+        self.length = length
+
+    def initialize(self):
+
```

Comment:
For this method, make a variable in your subsystem for if the elevator is moving, like `elevator_moving: bool`. please set it to true here, then false at `def end(self, interrupted)`. This is for the Limelights to know when to stop sending odometry data to the robot.